scripts/install/install_macos.sh

```diff
@@ -465,21 +468,28 @@ ask_clean_install() {
   fi
 
   if [ -f "$SUPERVISOR_YML_PATH" ]; then
-    command printf "${indent}An installation already exists. Would you like to do a clean install? $(prompt n)"
-    read -r clean_install_response
-    clean_install_response=$(echo "$clean_install_response" | tr '[:upper:]' '[:lower:]')
-    case $clean_install_response in
-    y | yes)
-      increase_indent
-      success "Doing clean install!"
-      decrease_indent
+    # Check for default config file hash
+    cfg_file_hash=$(sha256sum "$SUPERVISOR_YML_PATH" | awk '{print $1}')
+    if [ "$cfg_file_hash" == "$DEFAULT_SUPERVISOR_CFG_HASH"]; then
```

Comment:
Shellcheck fails here. Will also need to be fixed on the linux script.

```
In scripts/install/install_macos.sh line 473:
    if [ "$cfg_file_hash" == "$DEFAULT_SUPERVISOR_CFG_HASH"]; then
    ^-- SC1009 (info): The mentioned syntax error was in this if expression.
       ^-- SC1073 (error): Couldn't parse this test expression. Fix to allow more checks.
                                                            ^-- SC1020 (error): You need a space before the ].
                                                            ^-- SC1072 (error): Missing space before ]. Fix any mentioned problems and try again.
```

Reply:
Also changed those conditionals to use `=` instead of `==` since shellcheck was giving me warnings for that: https://www.shellcheck.net/wiki/SC3014